src/Uno.UI/UI/Xaml/Controls/ListView/ListViewItem.cs

```diff
@@ -26,5 +26,14 @@ protected override AutomationPeer OnCreateAutomationPeer()
 		{
 			return new ListViewItemAutomationPeer(this);
 		}
+
+		private protected override void OnLoaded()
+		{
+			base.OnLoaded();
+			if (Selector is ListView lv)
+			{
+				ApplyMultiSelectState(lv.IsSelectionMultiple);
+			}
```

Comment:
Thanks for the change! We'll need to also add a runtime test to ensure that we're indeed showing the check properly.

You could make a test that validates the active state, or that checks the visibility of a child container "MultiSelectCheck".

Reply:
I already did it with the[ DragDrop_ListView](https://github.com/unoplatform/uno/blob/fef77bcbb532d7713df799f7ff4ea031d6350413/src/SamplesApp/UITests.Shared/Windows_UI_Xaml/DragAndDrop/DragDrop_ListView.xaml#L20).
It works the same.
Do I still need to create a new test page for this?

Reply:
![DragDrop_ListView](https://github.com/unoplatform/uno/assets/69314237/0c8e373b-0ba7-466f-9a32-7634dfa0b599)

Reply:
Thanks. This is a sample. A runtime test is an automated test that will run in the CI to ensure that we do not regress what you've fixed with this PR. 

You can find some examples here:
https://github.com/unoplatform/uno/blob/87a7954fa107cf75ec2bf964d16b2d69f285ad52/src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListViewBase.cs#L255

And [the documentation here](https://platform.uno/docs/articles/uno-development/creating-runtime-tests.html).

Reply:
Added

Reply:
ListViewItem
![image](https://github.com/unoplatform/uno/assets/69314237/2c6aa516-9799-4872-887f-f3d80190949b)

Is I need to do the same for GridViewItem?

Reply:
The test you added is good. Thanks!